packages/build/tests/plugins/tests.js

```diff
@@ -306,23 +307,20 @@ test('--saveConfig creates netlify.toml if it does not exist', async (t) => {
   const fixtureDir = `${FIXTURES_DIR}/config_save_empty`
   const configPath = `${fixtureDir}/netlify.toml`
   await del(configPath)
+  const { address, stopServer } = await startDeployServer()
   try {
-    const { address, stopServer } = await startDeployServer()
-    try {
-      await runFixture(t, 'config_save_empty', {
-        flags: {
-          buildbotServerSocket: address,
-          saveConfig: true,
-          context: 'production',
-          branch: 'main',
-          defaultConfig: { plugins: [{ package: './plugin.js' }] },
-        },
-      })
-    } finally {
-      await stopServer()
-    }
+    await runFixture(t, 'config_save_empty', {
+      flags: {
+        buildbotServerSocket: address,
+        saveConfig: true,
+        context: 'production',
+        branch: 'main',
+        defaultConfig: { plugins: [{ package: './plugin.js' }] },
+      },
+    })
+    t.false(await pathExists(configPath))
   } finally {
-    await del(configPath)
```

Comment:
The `del()` calls in those two tests are not necessary anymore due to this PR's change.